### PR TITLE
IGAPP-1257: Add missing focus

### PR DESCRIPTION
**Pull request by f1sh1918**
Just use tab to navigate through the page

This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by LeandraH — APPROVED** [review]
Tested in Firefox, works. I see now what the report meant about how hard it is to navigate our page with only a keyboard :/

File: release-notes/unreleased/IGAPP-1257-Add-missing-focus.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+issue_key: IGAPP-1172
```
Comment (sarahsporck) [inline release-notes/unreleased/IGAPP-1257-Add-missing-focus.yml:1]:
```suggestion
issue_key: IGAPP-1257
```

**Review by sarahsporck — APPROVED** [review]
Tested in Safari.
I guess you agreed on the default highlight color? In the issue another color was adressed.

**Comment by f1sh1918** [comment]
> 

@sarahsporck the textSecondary color was addressed in the issue and should be used here? What do you mean by default color the greenish one?

**Comment by sarahsporck** [comment]
> > 
> 
> @sarahsporck the textSecondary color was addressed in the issue and should be used here! What do you mean by default color the greenish one?

For me a blieish color is used, so I assumed it is the default one o.o

**Comment by f1sh1918** [comment]
> 

image
this is #585858

**Comment by LeandraH** [comment]
Huh. This is what it looks like on my machine (in Firefox), and the only thing I can focus on while tabbing is the language selector.
image

**Comment by f1sh1918** [comment]
> Huh. This is what it looks like on my machine (in Firefox), and the only thing I can focus on while tabbing is the language selector. image

I fixed this by last commit

**Enable tap navigation in browser**

mac & firefox
https://stackoverflow.com/a/64151235

safari
https://osxdaily.com/2011/08/14/enable-tab-key-navigation-safari/

**Comment by LeandraH** [comment]
True, I see full on gray frames now, and did have to enable the tabbing in the system settings, thanks for the hint :)

**Pull request merged**